src/compute/src/extensions/arrange.rs

```diff
@@ -157,9 +130,10 @@ where
     G: Scope,
     K: Data + Columnation,
     G::Timestamp: Lattice,
-    R: Semigroup,
+    R: Data + Semigroup + Ord,
```

Comment:
This stems from `BatchContainer::ReadItem<'_>: Ord` and hence imposes it on all implementations for `BatchContainer`. Since we're using a batch container for storing diffs, we need the diff to be `Ord`.